<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<style type="text/css">
.txt { white-space:nowrap; }
.f0 { font-family:sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff1; src: url("1.otf"); }
.f1 { font-family:ff1,serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff3; src: url("3.otf"); }
.f2 { font-family:ff3,serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff6; src: url("6.otf"); }
.f3 { font-family:ff6,serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff2; src: url("2.otf"); }
.f4 { font-family:ff2,serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff10; src: url("10.otf"); }
.f5 { font-family:ff10,sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff4; src: url("4.otf"); }
.f6 { font-family:ff4,serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff5; src: url("5.otf"); }
.f7 { font-family:ff5,sans-serif; font-weight:normal; font-style:normal; }
</style>
</head>
<body>
<img style="position:absolute; left:0px; top:0px;" width="612" height="792" src="page16.png"">
<div class="txt" style="position:absolute; left:78px; top:36px;"><span class="f6" style="font-size:7px;vertical-align:baseline;color:rgba(145,148,150,1);">Research article</span></div>
<div class="txt" style="position:absolute; left:444px; top:36px;"><span class="f6" style="font-size:7px;vertical-align:baseline;color:rgba(23,163,216,1);">Microbiology and Infectious Disease</span></div>
<div class="txt" style="position:absolute; left:168px; top:53px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">and its concentration was determined spectrophotometrically in a NanoDrop Lite Spectrophotome-</span></div>
<div class="txt" style="position:absolute; left:168px; top:66px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">ter (Thermo Scientific). qRT-PCR was performed using an iQ 5 Multicolor Real-Time PCR Detection</span></div>
<div class="txt" style="position:absolute; left:168px; top:78px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">System (Bio-Rad) with qPCRBIO SyGreen Mix fluorescein (PCR BIOSYSTEMS). Master mixes were</span></div>
<div class="txt" style="position:absolute; left:168px; top:91px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">prepared as recommended by the manufacturer, with qRT-PCR primers listed in </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">Supplementary file</span></div>
<div class="txt" style="position:absolute; left:168px; top:104px;"><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">2 </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">– Supplementary Table 3. For normalization, either </span><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">gyrA </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">or </span><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">hfq </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">was used as internal standards.</span></div>
<div class="txt" style="position:absolute; left:168px; top:125px;"><span class="f4" style="font-size:11px;vertical-align:baseline;color:rgba(0,0,0,1);">Transposon mutagenesis of </span><span class="f2" style="font-size:11px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae</span></div>
<div class="txt" style="position:absolute; left:168px; top:140px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">Transposon mutagenesis was performed by conjugation of </span><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">co969 (recipient strain) with</span></div>
<div class="txt" style="position:absolute; left:168px; top:153px;"><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">E. coli </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">SM10 </span><span class="f5" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">l </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">pir (pSC189) (</span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">Chiang and Rubin, 2002</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">) (donor strain), as described by </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">Do¨ rr et al.,</span></div>
<div class="txt" style="position:absolute; left:168px; top:166px;"><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">2016</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">. Transposon mutants were selected on LB plates (10 g/l NaCl) containing both Sm(200 </span><span class="f5" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">m</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">g/ml)</span></div>
<div class="txt" style="position:absolute; left:168px; top:178px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">and Km (50 </span><span class="f5" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">m</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">g/ml). For the screen for biofilm repressors at 22˚C, 459 out of 11,000 screened</span></div>
<div class="txt" style="position:absolute; left:168px; top:191px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">mutants were selected due to their rugose colony morphology at 22˚C. To identify the genes inter-</span></div>
<div class="txt" style="position:absolute; left:168px; top:203px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">rupted by the transposon, the selected mutants were pooled together and transposon insertion</span></div>
<div class="txt" style="position:absolute; left:168px; top:216px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">sequencing (Tn-seq) was performed as described previously (</span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">Chao et al., 2013</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">). In brief, the chro-</span></div>
<div class="txt" style="position:absolute; left:168px; top:229px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">mosomal DNA of the pooled mutants was purified and sheared by sonication (Covaris Sonicator</span></div>
<div class="txt" style="position:absolute; left:168px; top:241px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">E220). The overhanging ends were blunted using a blunting enzyme kit (NEB) and an A-tail was</span></div>
<div class="txt" style="position:absolute; left:168px; top:254px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">added with Taq DNA polymerase (NEB). Then, Illumina adaptors were ligated (with T4 DNA ligase,</span></div>
<div class="txt" style="position:absolute; left:168px; top:266px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">NEB) and the genomic DNA-transposon junctions were amplified (with Phusion High-Fidelity DNA</span></div>
<div class="txt" style="position:absolute; left:168px; top:279px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">polymerase, NEB). Sequencing was performed using an Illumina MiSeq benchtop sequencer (Illu-</span></div>
<div class="txt" style="position:absolute; left:168px; top:291px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">mina, San Diego, CA). Data analysis for determination of the TA dinucleotide insertion sites was con-</span></div>
<div class="txt" style="position:absolute; left:168px; top:304px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">ducted as described previously (</span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">Chao et al., 2013</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">; </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">Pritchard et al., 2014</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">). For visualization of</span></div>
<div class="txt" style="position:absolute; left:168px; top:316px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">transposon insertion profiles, Sanger Artemis Genome Browser and Annotation tool were used</span></div>
<div class="txt" style="position:absolute; left:168px; top:329px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">(</span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">Rutherford et al., 2000</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">).</span></div>
<div class="txt" style="position:absolute; left:180px; top:341px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">To avoid selecting false-positive candidates with a temperature-independent effect on biofilm for-</span></div>
<div class="txt" style="position:absolute; left:168px; top:353px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">mation, another transposon mutagenesis was performed in parallel, by conjugation of the constitu-</span></div>
<div class="txt" style="position:absolute; left:168px; top:366px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">tively smooth </span><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">C6706 with </span><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">E. coli </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">SM10 </span><span class="f5" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">l </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">pir (pSC189). Out of 63,000 screened mutants,</span></div>
<div class="txt" style="position:absolute; left:168px; top:378px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">618 formed rugose colonies at 37˚C. Identification of the genes interrupted by the transposon was</span></div>
<div class="txt" style="position:absolute; left:168px; top:391px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">performed using the approach described above. Genes that were hit in both transposon screens</span></div>
<div class="txt" style="position:absolute; left:168px; top:403px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">were filtered out during the selection of candidates for further analysis. Higher number of reads per</span></div>
<div class="txt" style="position:absolute; left:168px; top:416px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">clone was used as a parameter for candidate selection.</span></div>
<div class="txt" style="position:absolute; left:180px; top:428px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">A third transposon mutagenesis was performed by conjugation of </span><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">co969 with </span><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">E. coli</span></div>
<div class="txt" style="position:absolute; left:168px; top:440px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">SM10 </span><span class="f5" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">l </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">pir (pSC189), for selection of mutants with smooth colony morphology at 37˚C to screen for</span></div>
<div class="txt" style="position:absolute; left:168px; top:453px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">a potential BipA-specific protease. Out of 19,000 screened mutants, 313 were selected. Identifica-</span></div>
<div class="txt" style="position:absolute; left:168px; top:465px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">tion of the genes interrupted by the transposon was performed as described above.</span></div>
<div class="txt" style="position:absolute; left:180px; top:477px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">The data for this study have been deposited in the ENA at EMBL-EBI under accession number</span></div>
<div class="txt" style="position:absolute; left:168px; top:489px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">PRJEB42487 (https://www.ebi.ac.uk/ena/browser/view/PRJEB42487).</span></div>
<div class="txt" style="position:absolute; left:168px; top:511px;"><span class="f4" style="font-size:11px;vertical-align:baseline;color:rgba(0,0,0,1);">Western blotting of </span><span class="f2" style="font-size:11px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f4" style="font-size:11px;vertical-align:baseline;color:rgba(0,0,0,1);">colonies</span></div>
<div class="txt" style="position:absolute; left:168px; top:526px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">For Western blotting, two colonies were collected per sample, resuspended in buffer (20 mM Tris</span></div>
<div class="txt" style="position:absolute; left:168px; top:538px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">pH 7.5, 200 mM NaCl, 5 mM EDTA and protease inhibitor [cOmplete protease inhibitor cocktail tab-</span></div>
<div class="txt" style="position:absolute; left:168px; top:551px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">lets; Sigma Aldrich]), disrupted on a bead beater (Mini beadbeater, Biospec Products; 4˚C, maximum</span></div>
<div class="txt" style="position:absolute; left:168px; top:564px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">speed for 1 min 15 s), and the total protein concentration of the samples was determined by Brad-</span></div>
<div class="txt" style="position:absolute; left:168px; top:576px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">ford assay (</span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(109,109,112,1);">Bradford, 1976</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">) with Protein Assay Dye Reagent Concentrate (Bio-Rad). Samples were</span></div>
<div class="txt" style="position:absolute; left:168px; top:589px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">normalized by total protein content and run in a 10% SDS-PAGE (sodium dodecyl sulfate polyacryl-</span></div>
<div class="txt" style="position:absolute; left:168px; top:601px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">amide gel electrophoresis) gel for protein separation. Proteins were then transferred to an Immobi-</span></div>
<div class="txt" style="position:absolute; left:168px; top:614px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">lon-P transfer membrane (Millipore) using a Trans-Blot Turbo Transfer system (Bio-Rad). After</span></div>
<div class="txt" style="position:absolute; left:168px; top:626px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">blocking the membrane at room temperature for 2 hr with TBS-T buffer (24.2 g Trizma, 87.6 g NaCl</span></div>
<div class="txt" style="position:absolute; left:168px; top:639px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">in 1 l H</span><span class="f1" style="font-size:5px;vertical-align:baseline;color:rgba(0,0,0,1);">2</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">O, pH 7.5, 0.1% (v/v) Tween-20) containing 5% (w/v) milk, the membrane was incubated at</span></div>
<div class="txt" style="position:absolute; left:168px; top:651px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">4˚C o/n with the monoclonal anti-flag M2 primary antibody (Sigma Aldrich) in TBS-T buffer contain-</span></div>
<div class="txt" style="position:absolute; left:168px; top:664px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">ing 5% (w/v) milk, washed with TBS-T buffer, incubated at RT for 1 hr with </span><span class="f5" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">a</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">-mouse secondary anti-</span></div>
<div class="txt" style="position:absolute; left:168px; top:677px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">body in TBS-T buffer and again washed with TBS-T buffer. BipA-flag was detected by addition of</span></div>
<div class="txt" style="position:absolute; left:168px; top:689px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">SuperSignal West Pico Chemiluminiscence substrate (Thermo Scientific) and detection of the signal</span></div>
<div class="txt" style="position:absolute; left:168px; top:702px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">using a LAS4000 (Fujifilm).</span></div>
<div class="txt" style="position:absolute; left:37px; top:748px;"><span class="f6" style="font-size:7px;vertical-align:baseline;color:rgba(109,109,112,1);">del Peso Santos </span><span class="f7" style="font-size:7px;vertical-align:baseline;color:rgba(109,109,112,1);">et al</span><span class="f6" style="font-size:7px;vertical-align:baseline;color:rgba(109,109,112,1);">. eLife 2021;10:e60607. DOI: </span><a href="https://doi.org/10.7554/eLife.60607"><span class="f6" style="font-size:7px;vertical-align:baseline;color:rgba(23,163,216,1);">https://doi.org/10.7554/eLife.60607</span></a></div>
<div class="txt" style="position:absolute; left:545px; top:748px;"><span class="f6" style="font-size:7px;vertical-align:baseline;color:rgba(101,104,108,1);">16 of 23</span></div>
</body>
</html>
