<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<style type="text/css">
.txt { white-space:nowrap; }
.f0 { font-family:sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff8; src: url("8.otf"); }
.f1 { font-family:ff8,serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff10; src: url("10.otf"); }
.f2 { font-family:ff10,serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff13; src: url("13.otf"); }
.f3 { font-family:ff13,sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff11; src: url("11.otf"); }
.f4 { font-family:ff11,sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff4; src: url("4.otf"); }
.f5 { font-family:ff4,sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff7; src: url("7.otf"); }
.f6 { font-family:ff7,sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff14; src: url("14.otf"); }
.f7 { font-family:ff14,sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff18; src: url("18.otf"); }
.f8 { font-family:ff18,sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff19; src: url("19.otf"); }
.f9 { font-family:ff19,sans-serif; font-weight:normal; font-style:normal; }
@font-face { font-family: ff2; src: url("2.otf"); }
.f10 { font-family:ff2,sans-serif; font-weight:normal; font-style:normal; }
</style>
</head>
<body>
<img style="position:absolute; left:0px; top:0px;" width="584" height="782" src="page7.png"">
<div class="txt" style="position:absolute; left:47px; top:52px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">These results indicate that chemotaxis itself is not required for</span></div>
<div class="txt" style="position:absolute; left:38px; top:62px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">biofilm dispersal, but the chemotaxis machinery facilitates ran-</span></div>
<div class="txt" style="position:absolute; left:38px; top:72px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">dom reorientation events that allow </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">cells to navigate</span></div>
<div class="txt" style="position:absolute; left:38px; top:82px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">a porous biofilm matrix. The same nonchemotactic mutants used</span></div>
<div class="txt" style="position:absolute; left:38px; top:92px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">here exhibit stark differences in competition experiments in an-</span></div>
<div class="txt" style="position:absolute; left:38px; top:102px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">imal models of cholera infection, demonstrating that their dif-</span></div>
<div class="txt" style="position:absolute; left:38px; top:112px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">ferences in motility, and possibly also in biofilm dispersal</span></div>
<div class="txt" style="position:absolute; left:38px; top:123px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">capability, are pertinent to colonization (34).</span></div>
<div class="txt" style="position:absolute; left:47px; top:132px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">Finally, we determined whether the ability to locomote was</span></div>
<div class="txt" style="position:absolute; left:38px; top:143px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">required for biofilm dispersal or, by contrast, if nonmotile cells</span></div>
<div class="txt" style="position:absolute; left:38px; top:153px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">could escape the digested matrix via Brownian motion. As</span></div>
<div class="txt" style="position:absolute; left:38px; top:163px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">mentioned above, we could not simply study dispersal of non-</span></div>
<div class="txt" style="position:absolute; left:38px; top:173px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">flagellated and nonmotile mutants because of their confounding</span></div>
<div class="txt" style="position:absolute; left:38px; top:183px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">surface attachment defects and feedback on biofilm regulatory</span></div>
<div class="txt" style="position:absolute; left:38px; top:193px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">components (37, 38). To circumvent this problem, we used</span></div>
<div class="txt" style="position:absolute; left:38px; top:201px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">phenamil, an inhibitor of the Na</span><span class="f1" style="font-size:5px;vertical-align:super;color:rgba(0,0,0,1);">+</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">-driven </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">flagellar</span></div>
<div class="txt" style="position:absolute; left:38px; top:213px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">motor, which dramatically reduced planktonic cell motility, as</span></div>
<div class="txt" style="position:absolute; left:38px; top:223px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">expected (</span><a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2021166117/-/DCSupplemental"><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,255,1);">SI Appendix</span></a><a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2021166117/-/DCSupplemental"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,255,1);">, Fig. S4</span></a><a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2021166117/-/DCSupplemental"><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,255,1);">B</span></a><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">) (39). To assess the role of</span></div>
<div class="txt" style="position:absolute; left:38px; top:233px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">swimming motility in biofilm dispersal, we first allowed WT </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V.</span></div>
<div class="txt" style="position:absolute; left:38px; top:243px;"><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">cells to undergo biofilm formation for 5 h, at which</span></div>
<div class="txt" style="position:absolute; left:38px; top:253px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">point we perfused DMSO or phenamil into the incubation</span></div>
<div class="txt" style="position:absolute; left:38px; top:263px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">chamber (Fig. 4</span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">G</span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">). Following phenamil treatment, the WT</span></div>
<div class="txt" style="position:absolute; left:38px; top:273px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">strain displayed a dispersal defect nearly identical to that of the</span></div>
<div class="txt" style="position:absolute; left:38px; top:283px;"><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">Δ</span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">cheY3 </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">mutant. In addition, phenamil treatment of the </span><span class="f3" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">Δ</span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">cheY3</span></div>
<div class="txt" style="position:absolute; left:38px; top:293px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">mutant did not further impair its biofilm dispersal. Together,</span></div>
<div class="txt" style="position:absolute; left:38px; top:303px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">these results demonstrate that swimming motility is crucial for </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V.</span></div>
<div class="txt" style="position:absolute; left:38px; top:313px;"><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">biofilm dispersal, and that an inability to reorient is as</span></div>
<div class="txt" style="position:absolute; left:38px; top:323px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">detrimental to dispersal as a complete lack of flagellar motility.</span></div>
<div class="txt" style="position:absolute; left:38px; top:341px;"><span class="f4" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">Discussion</span></div>
<div class="txt" style="position:absolute; left:38px; top:353px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">Previous studies provided initial clues concerning the regulation</span></div>
<div class="txt" style="position:absolute; left:38px; top:363px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">of </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">biofilm dispersal: quorum sensing, the process of</span></div>
<div class="txt" style="position:absolute; left:38px; top:373px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">cell-to-cell communication that bacteria use to orchestrate col-</span></div>
<div class="txt" style="position:absolute; left:38px; top:383px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">lective behaviors, bile salts, and starvation signals promote dis-</span></div>
<div class="txt" style="position:absolute; left:38px; top:393px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">persal (18, 19, 40, 41). However, other components, and</span></div>
<div class="txt" style="position:absolute; left:38px; top:404px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">specifically those that enable the cells to exit the biofilms,</span></div>
<div class="txt" style="position:absolute; left:38px; top:414px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">remained largely unknown. In this study, we developed a high-</span></div>
<div class="txt" style="position:absolute; left:38px; top:424px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">content imaging screen that allowed us to identify components</span></div>
<div class="txt" style="position:absolute; left:38px; top:434px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">required for </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">biofilm dispersal. We categorized the</span></div>
<div class="txt" style="position:absolute; left:38px; top:444px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">identified components into three classes: signal transduction,</span></div>
<div class="txt" style="position:absolute; left:38px; top:454px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">matrix disassembly, and cell motility. We propose that the three</span></div>
<div class="txt" style="position:absolute; left:38px; top:464px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">functional categories represent the chronological steps required</span></div>
<div class="txt" style="position:absolute; left:38px; top:474px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">for the disassembly of a biofilm. First, the stimuli that activate</span></div>
<div class="txt" style="position:absolute; left:38px; top:484px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">dispersal must accumulate. Subsequently, the gene expression</span></div>
<div class="txt" style="position:absolute; left:38px; top:494px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">pattern established by detection of these stimuli must repress</span></div>
<div class="txt" style="position:absolute; left:38px; top:504px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">biofilm matrix production and activate production of enzymes</span></div>
<div class="txt" style="position:absolute; left:38px; top:514px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">required to digest the biofilm matrix. Finally, cells must escape</span></div>
<div class="txt" style="position:absolute; left:38px; top:524px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">through the partially digested, porous matrix, which requires</span></div>
<div class="txt" style="position:absolute; left:38px; top:534px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">changes in the direction of movement.</span></div>
<div class="txt" style="position:absolute; left:47px; top:544px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">Here we show that the DbfS-DbfR two-component system</span></div>
<div class="txt" style="position:absolute; left:38px; top:554px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">controls matrix production. While not yet tested explicitly, it is</span></div>
<div class="txt" style="position:absolute; left:38px; top:564px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">possible that DbfS-DbfR also orchestrates the initiation of ma-</span></div>
<div class="txt" style="position:absolute; left:38px; top:574px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">trix digestion and the launch of motility. Together, these steps</span></div>
<div class="txt" style="position:absolute; left:38px; top:584px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">ensure that when environmental conditions are appropriate, </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V.</span></div>
<div class="txt" style="position:absolute; left:38px; top:594px;"><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">cells can exit the sessile lifestyle and disseminate to new</span></div>
<div class="txt" style="position:absolute; left:38px; top:604px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">terrain that is ripe for biofilm formation or, alternatively during</span></div>
<div class="txt" style="position:absolute; left:38px; top:614px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">disease, to a new host. One can now imagine targeting the</span></div>
<div class="txt" style="position:absolute; left:38px; top:624px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">functions identified in this work for small-molecule disruption of</span></div>
<div class="txt" style="position:absolute; left:38px; top:635px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">the </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">biofilm lifecycle, possibly guiding the develop-</span></div>
<div class="txt" style="position:absolute; left:38px; top:645px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">ment of treatments to reduce the duration of </span><span class="f2" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">infec-</span></div>
<div class="txt" style="position:absolute; left:38px; top:655px;"><span class="f1" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">tion or to prevent transmission.</span></div>
<div class="txt" style="position:absolute; left:38px; top:672px;"><span class="f4" style="font-size:8px;vertical-align:baseline;color:rgba(0,0,0,1);">Materials and Methods</span></div>
<div class="txt" style="position:absolute; left:38px; top:685px;"><span class="f4" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Bacterial Strains and Reagents. </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">The </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">parent strain used in this</span></div>
<div class="txt" style="position:absolute; left:38px; top:694px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">study was WT O1 El Tor biotype C6706str2. Antibiotics were used at the</span></div>
<div class="txt" style="position:absolute; left:38px; top:703px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">following concentrations: polymyxin B, 50 </span><span class="f3" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">μ</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">g/mL; kanamycin, 50 </span><span class="f3" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">μ</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">g/mL;</span></div>
<div class="txt" style="position:absolute; left:38px; top:712px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">spectinomycin, 200 </span><span class="f3" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">μ</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">g/mL; and chloramphenicol, 1 </span><span class="f3" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">μ</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">g/mL Strains were</span></div>
<div class="txt" style="position:absolute; left:38px; top:721px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">propagated in lysogeny broth (LB) supplemented with 1.5% agar or in liquid</span></div>
<div class="txt" style="position:absolute; left:296px; top:52px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">LB with shaking at 30 °C. All strains used in this work are reported in </span><a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2021166117/-/DCSupplemental"><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,255,1);">SI</span></a></div>
<div class="txt" style="position:absolute; left:296px; top:61px;"><a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2021166117/-/DCSupplemental"><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,255,1);">Appendix</span></a><a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2021166117/-/DCSupplemental"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,255,1);">, Table </span></a><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,255,1);">S1. Unless stated otherwise, exogenous compounds were</span></div>
<div class="txt" style="position:absolute; left:296px; top:70px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">added from the onset of biofilm initiation. The antimicrobial peptide C18G</span></div>
<div class="txt" style="position:absolute; left:296px; top:79px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">(VWR) was added at 5 </span><span class="f3" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">μ</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">g/mL. Phenamil (Sigma-Aldrich) was prepared in</span></div>
<div class="txt" style="position:absolute; left:296px; top:88px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">DMSO and added at 5 h post-biofilm inoculation to a final concentration of</span></div>
<div class="txt" style="position:absolute; left:296px; top:97px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">50 </span><span class="f3" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">μ</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">M. </span><span class="f5" style="font-size:5px;vertical-align:baseline;color:rgba(0,0,0,1);">L</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">-arabinose (Sigma-Aldrich) was prepared in water and added</span></div>
<div class="txt" style="position:absolute; left:296px; top:106px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">at 0.2%.</span></div>
<div class="txt" style="position:absolute; left:296px; top:125px;"><span class="f4" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">DNA Manipulation and Strain Construction. </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">To produce linear DNA fragments</span></div>
<div class="txt" style="position:absolute; left:296px; top:134px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">for natural transformations, splicing overlap extension PCR was performed</span></div>
<div class="txt" style="position:absolute; left:296px; top:143px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">using iProof polymerase (Bio-Rad) to combine DNA pieces. Primers and gene</span></div>
<div class="txt" style="position:absolute; left:296px; top:152px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">fragments used in this study are reported in </span><a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2021166117/-/DCSupplemental"><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,255,1);">SI Appendix</span></a><a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2021166117/-/DCSupplemental"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,255,1);">, Table </span></a><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,255,1);">S2. In all</span></div>
<div class="txt" style="position:absolute; left:296px; top:160px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">cases, </span><span class="f7" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">∼</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">3 kb of upstream and downstream flanking regions of homology</span></div>
<div class="txt" style="position:absolute; left:296px; top:170px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">were generated by PCR from </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">genomic DNA and were included to</span></div>
<div class="txt" style="position:absolute; left:296px; top:179px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">ensure high chromosomal integration frequency. DNA fragments that were</span></div>
<div class="txt" style="position:absolute; left:296px; top:188px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">not native to </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">were synthesized as g-blocks (IDT).</span></div>
<div class="txt" style="position:absolute; left:305px; top:198px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">All </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">strains generated in this work were constructed by</span></div>
<div class="txt" style="position:absolute; left:296px; top:207px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">replacing genomic DNA with DNA introduced by natural transformation as</span></div>
<div class="txt" style="position:absolute; left:296px; top:216px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">described previously (18, 42). The neutral </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">vc1807 </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">locus was used as the site of</span></div>
<div class="txt" style="position:absolute; left:296px; top:225px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">introduction of the gene encoding the antibiotic resistance cassette in the</span></div>
<div class="txt" style="position:absolute; left:296px; top:234px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">natural cotransformation procedure. The </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">vc1807 </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">locus also served as the site</span></div>
<div class="txt" style="position:absolute; left:296px; top:243px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">for the introduction of genes under study in chromosomal ectopic expres-</span></div>
<div class="txt" style="position:absolute; left:296px; top:252px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">sion analyses (42). PCR and Sanger sequencing were used to verify correct</span></div>
<div class="txt" style="position:absolute; left:296px; top:261px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">integration events. Genomic DNA from recombinant strains was used for</span></div>
<div class="txt" style="position:absolute; left:296px; top:269px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">future cotransformations and as templates for PCR to generate DNA frag-</span></div>
<div class="txt" style="position:absolute; left:296px; top:278px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">ments when necessary. Deletions were constructed in frame and eliminated</span></div>
<div class="txt" style="position:absolute; left:296px; top:287px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">the entire coding sequences. The exceptions were </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">mbaA</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">, </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">dbfS</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">, and </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">dbfR</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">,</span></div>
<div class="txt" style="position:absolute; left:296px; top:296px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">each of which overlaps with another gene in their operons. In these cases,</span></div>
<div class="txt" style="position:absolute; left:296px; top:305px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">portions of the genes were deleted ensuring that adjacent genes were not</span></div>
<div class="txt" style="position:absolute; left:296px; top:314px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">perturbed. For </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">tagA</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">, the first 103 base pairs, including the nucleotides</span></div>
<div class="txt" style="position:absolute; left:296px; top:323px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">specifying the start codon, were deleted. All strains constructed in this study</span></div>
<div class="txt" style="position:absolute; left:296px; top:332px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">were verified by sequencing at GENEWIZ.</span></div>
<div class="txt" style="position:absolute; left:296px; top:351px;"><span class="f4" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Microscopy and Mutant Screening. </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">The biofilm lifecycle was measured using</span></div>
<div class="txt" style="position:absolute; left:296px; top:360px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">time-lapse microscopy as described previously (18). All plots were gener-</span></div>
<div class="txt" style="position:absolute; left:296px; top:369px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">ated using ggplot2 in R. To generate the library of </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">V. cholerae </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">insertion</span></div>
<div class="txt" style="position:absolute; left:296px; top:378px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">mutants for the dispersal screen, the WT parent strain was mutagenized</span></div>
<div class="txt" style="position:absolute; left:296px; top:387px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">with Tn</span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">5 </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">as described previously (43). Mutants were selected by growth</span></div>
<div class="txt" style="position:absolute; left:296px; top:396px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">overnight on LB plates containing polymyxin B and kanamycin. The next day,</span></div>
<div class="txt" style="position:absolute; left:296px; top:405px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">mutant colonies were arrayed into 96-well plates containing 200 </span><span class="f3" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">μ</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">L of LB</span></div>
<div class="txt" style="position:absolute; left:296px; top:414px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">medium supplemented with polymyxin B and kanamycin using an auto-</span></div>
<div class="txt" style="position:absolute; left:296px; top:423px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">mated colony-picking robot (Molecular Devices). The arrayed cultures were</span></div>
<div class="txt" style="position:absolute; left:296px; top:432px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">grown in a plate-shaking incubator at 30 °C covered with Breathe-Easy</span></div>
<div class="txt" style="position:absolute; left:296px; top:441px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">membranes (Sigma-Aldrich) to minimize evaporation. After 16 h of</span></div>
<div class="txt" style="position:absolute; left:296px; top:450px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">growth, the arrayed cultures were diluted 1:200,000 into 96-well plates</span></div>
<div class="txt" style="position:absolute; left:296px; top:459px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">containing M9 medium supplemented with glucose and casamino acids.</span></div>
<div class="txt" style="position:absolute; left:296px; top:468px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Diluted cultures were incubated statically at 30 °C for 8 h (to achieve peak</span></div>
<div class="txt" style="position:absolute; left:296px; top:477px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">biofilm biomass), at which point images of each well were captured on a</span></div>
<div class="txt" style="position:absolute; left:296px; top:486px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Nikon Ti-E inverted microscope using transmitted-light bright-field illumi-</span></div>
<div class="txt" style="position:absolute; left:296px; top:494px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">nation, a 10</span><span class="f8" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">× </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Plan Fluor (NA 0.3) objective lens, and an Andor iXon 897</span></div>
<div class="txt" style="position:absolute; left:296px; top:504px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">EMCCD camera. Automated image acquisition was performed using NIS-</span></div>
<div class="txt" style="position:absolute; left:296px; top:513px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Elements software v5.11.02 and the NIS-Elements Jobs Module to acquire</span></div>
<div class="txt" style="position:absolute; left:296px; top:522px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">images at four positions within each well to account for heterogeneity</span></div>
<div class="txt" style="position:absolute; left:296px; top:531px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">within samples. The focal plane between wells was maintained using the</span></div>
<div class="txt" style="position:absolute; left:296px; top:540px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Nikon Perfect Focus system.</span></div>
<div class="txt" style="position:absolute; left:305px; top:550px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">After microscopy at the 8 h timepoint, 96-well plates were returned to the</span></div>
<div class="txt" style="position:absolute; left:296px; top:559px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">incubator. To assess biofilm dispersal, a second set of images of the same</span></div>
<div class="txt" style="position:absolute; left:296px; top:568px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">samples was acquired at 13 h postinoculation. Mutants that displayed biofilm</span></div>
<div class="txt" style="position:absolute; left:296px; top:577px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">growth at the 8 h timepoint but failed to disperse by the 13 h timepoint were</span></div>
<div class="txt" style="position:absolute; left:296px; top:585px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">subcultured, grown overnight, and subsequently reimaged using the time-</span></div>
<div class="txt" style="position:absolute; left:296px; top:594px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">lapse approach described above to assess their biofilm lifecycles in real time.</span></div>
<div class="txt" style="position:absolute; left:296px; top:603px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Mutants that exhibited biofilm dispersal defects after this reassessment step</span></div>
<div class="txt" style="position:absolute; left:296px; top:612px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">were analyzed for the locations of transposon insertions using arbitrary</span></div>
<div class="txt" style="position:absolute; left:296px; top:621px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">PCR (44).</span></div>
<div class="txt" style="position:absolute; left:296px; top:640px;"><span class="f9" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">vpsL-lux </span><span class="f4" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Transcription Assay. </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Three colonies of each strain to be analyzed were</span></div>
<div class="txt" style="position:absolute; left:296px; top:649px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">individually grown overnight in 200 </span><span class="f3" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">μ</span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">L of LB with shaking at 30 °C in a</span></div>
<div class="txt" style="position:absolute; left:296px; top:658px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">96-well plate covered with a Breathe-Easy membrane. The next morning, the</span></div>
<div class="txt" style="position:absolute; left:296px; top:667px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">cultures were diluted 1:5,000 into fresh M9 medium supplemented with</span></div>
<div class="txt" style="position:absolute; left:296px; top:676px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">glucose and casamino acids. The plates were placed in a BioTek Synergy</span></div>
<div class="txt" style="position:absolute; left:296px; top:685px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Neo2 Multi-Mode reader under static growth conditions at 30 °C. OD</span><span class="f5" style="font-size:4px;vertical-align:baseline;color:rgba(0,0,0,1);">600 </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">and</span></div>
<div class="txt" style="position:absolute; left:296px; top:694px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">bioluminescence from </span><span class="f6" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">vpsL-lux </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">were measured simultaneously at 15-min</span></div>
<div class="txt" style="position:absolute; left:296px; top:703px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">time intervals. The results were exported to R, and light values were divided</span></div>
<div class="txt" style="position:absolute; left:296px; top:712px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">by OD</span><span class="f5" style="font-size:4px;vertical-align:baseline;color:rgba(0,0,0,1);">600 </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">to produce relative light units (RLUs). Results from replicates were</span></div>
<div class="txt" style="position:absolute; left:296px; top:721px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">averaged and plotted using ggplot2 in R.</span></div>
<div class="txt" style="position:absolute; left:38px; top:746px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">Bridges et al.</span></div>
<div class="txt" style="position:absolute; left:356px; top:746px;"><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">PNAS </span><span class="f1" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">| </span><span class="f10" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">December 22, 2020 </span><span class="f1" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">| </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">vol. 117 </span><span class="f1" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">| </span><span class="f5" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">no. 51 </span><span class="f1" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">| </span><span class="f10" style="font-size:6px;vertical-align:baseline;color:rgba(0,0,0,1);">32645</span></div>
</body>
</html>
